examples/brute_force.py

```diff
@@ -288,11 +306,15 @@ def run_brute_force(
         worker.join()
 
     num_trials = sum(worker.num_trials for worker in workers)
+    env: CompilerEnv = make_env()
+    env.reset()
     print(
         f"completed {humanize.intcomma(num_trials)} of "
         f"{humanize.intcomma(expected_trial_count)} trials "
-        f"({num_trials / expected_trial_count:.3%})"
+        f"({num_trials / expected_trial_count:.3%}), best sequence",
+        ",".join([env.action_space.flags[i] for i in best_action_sequence]),
```

Comment:
```suggestion
        " ".join([env.action_space.flags[i] for i in best_action_sequence]),
```

Whitespace is a good separator for flags, as that is what you would use on the command line